bin/run

```diff
@@ -1,33 +1,30 @@
 #!/usr/bin/env node
 
-const { Config } = require('@oclif/config')
-
-const SdkQuery = require('../lib/commands/sdk/query')
+const { default: Help } = require('@oclif/plugin-help')
 
+const Help_showHelp = Help.prototype.showHelp;
```

Comment:
I don't want to firmly oppose to this monkey-patching. After all, it works and will do (as long as we don't update the Help plugin _and_ its API changes). But I think that the proper way to implement the solution to our need is to add our own help plugin.
https://oclif.io/docs/plugins

Reply:
I agree, we could also fork the existing one and adapt it for our needs.   

The first monkey patching was quite dangerous because any API change would break it. This one call the original method, catch the error, enhance it and then throw it.  
The only difference between the original method and the patched one is the error message.

I choose monkey patching because this time because it was safe and quick